web/src/app/modules/sugarloaf/components/smart/apply-yaml/apply-yaml.component.scss

```diff
@@ -24,8 +27,4 @@
       display: none;
     }
   }
-
```

Comment:
I think we need icon right margin here, actually it could be bigger than before to make it same as spacing for namespace/context dropdown icons.

Reply:
Can you be more specific about what is here?

Reply:
Sorry, apply-yaml icon right margin needs to be changed so there is identical spacing as for namespace/context dropdown icons.

Reply:
![Screen Shot 2021-04-21 at 11 43 01 AM](https://user-images.githubusercontent.com/34245594/115598077-3fdad700-a297-11eb-8880-8a71db798b00.png)